addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -129,6 +130,98 @@
         },
     });
 
+    /**
+     * This widget is responsible of loading the Vimeo video.
+     *
+     * Similarly to the YouTube implementation, the widget will trigger an event `change_slide` when
+     * the video is at its end, and `slide_completed` when the player is at 30 sec before the end of
+     * the video (30 sec before is considered as completed).
+     *
+     * See https://developer.vimeo.com/player/sdk/reference for all the API documentation.
+     */
+    var VideoPlayerVimeo = publicWidget.Widget.extend({
```

Comment:
Side linting question: isn't it better to have it in its own small file ? just to ease review / reading ?

Reply:
Ok ^^